frontend/src/router.tsx

```diff
@@ -1,31 +1,40 @@
-import React, { useLayoutEffect } from 'react';
+import React, { useLayoutEffect, lazy, Suspense } from 'react';
 import { Switch, Redirect, useLocation } from 'react-router-dom';
-import MapView from './features/properties/map/MapView';
-import AccessRequestPage from './features/admin/access-request/AccessRequestPage';
-import EditUserPage from './features/admin/edit-user/EditUserPage';
 import Login from './features/account/Login';
 import AccessDenied from './pages/401/AccessDenied';
-import { ManageUsers } from 'features/admin/users/ManageUsers';
-import ManageAccessRequests from 'features/admin/access/ManageAccessRequests';
+
 import { Claims } from 'constants/claims';
 import AppRoute from 'utils/AppRoute';
 import PublicLayout from 'layouts/PublicLayout';
 import AuthLayout from 'layouts/AuthLayout';
 import Test from 'pages/Test.ignore';
-import { PropertyListView } from 'features/properties/list';
+
 import { NotFoundPage } from 'pages/404/NotFoundPage';
-import {
-  ProjectDisposalSubmitted,
-  ProjectDisposalExemptionSubmitted,
-} from 'features/projects/dispose';
-import { ProjectListView, ProjectApprovalRequestListView } from 'features/projects/list';
 import { LogoutPage } from 'features/account/Logout';
-import { ProjectRouter } from 'features/projects/common';
-import { ProjectDisposeView } from 'features/projects/dispose';
-import SplReportContainer from 'features/splReports/containers/SplReportContainer';
-import ManageAgencies from 'features/admin/agencies/ManageAgencies';
-import EditAgencyPage from 'features/admin/agencies/EditAgencyPage';
 import { IENotSupportedPage } from 'features/account/IENotSupportedPage';
+import FilterBackdrop from 'components/maps/leaflet/FilterBackdrop';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import { Col } from 'react-bootstrap';
+
+const MapView = lazy(() => import('./features/properties/map/MapView'));
```

Comment:
lazy load all components sitting on an application route. This dramatically reduces the size of our initial js bundle.